Boardfiles/nucleol552zeq/Core/Src/main.cpp

```diff
@@ -218,6 +219,13 @@ void SystemClock_Config(void)
 
 /* USER CODE BEGIN 4 */
 
+int __io_putchar(int ch)

```

Comment:
I saw two copies of this code, maybe we only need one of them? Deleting the one in this file?